written/week1.md

```diff
@@ -0,0 +1,30 @@
+# Written problems - week 1
+
+This problem set covers lectures 1 through 3.
+
+## Lectures
+
+1. [Introduction](https://github.com/jlpteaching/ECS154B/blob/master/lecture%20notes/Lecture-1.pdf)
+2. [Technology](https://github.com/jlpteaching/ECS154B/blob/master/lecture%20notes/Lecture-2.pdf)
+3. [More technology](https://github.com/jlpteaching/ECS154B/blob/master/lecture%20notes/Lecture-3.pdf)
+
+## Topics covered
+
+* Computer architecture overview
+* History of computing
+  * Moore's law
+  * Dennard scaling
+* CMOS devices
+  * Energy
+  * Power
+* Trends in computing
+
+## Problems
+
+1. What is the Iron Law?
```

Comment:
This is going to be covered in Performance Lecture, ~#8 I think.

Reply:
Hmm, it was covered in the Turing Lecture too. Is that fair game, or would you rather wait until you officially cover it?